pages/docs/getting-started/event-driven-architectures.md

```diff
@@ -41,15 +41,15 @@ A subscriber (a.k.a. _consumer_) is an application that connects to the _broker_
 
 ### Message
 
-A message is a piece of information that's sent by the publishers to the broker, and received by all the interested subscribers. The content of the message can be anything, and they are frequently catalogued as _events_ and _commands_. As you saw above, _events_ communicate a fact that occurred. Instead, _commands_ are very much like _requests_ in REST APIs: They tell the subscribers "do this."
+A message is a piece of information that's sent by the publishers to the broker, and received by all the interested subscribers. The content of the message can be anything, and they are frequently catalogued as _events_ and _commands_. As you saw above, _events_ communicate a fact that occurred. On the other hand, _commands_ are very much like _requests_ in REST APIs: They tell the subscribers "do this."
 
-**Technically speaking, _events_ and _commands_ are the same. The only difference is in their semantics.**
+**To be precise, _events_ and _commands_ share the same structure, but differ conceptually.**
```

Comment:
```suggestion
**To be precise, _events_ and _commands_ share the same structure but differ conceptually.**
```

Reply:
not a native speaker 😅  but Grammarly (my personal assistant) 😆 doesn't complain about lack of a comma

Reply:
Comma is not needed there afaik.

Reply:
oh, sorry, what I meant, Grammarly says it is ok with and without